ex_feels/assets/css/app.scss

```diff
@@ -1,4 +1,7 @@
+@import url(//fonts.googleapis.com/css?family=Open+Sans:400,700);
```

Comment:
Lets not depend on a network request to get our fonts. If the default font isn't desirable, lets change it via [bulma](https://bulma.io/)

Reply:
lol @ this introduction video. this reminds me of the git vs. svn video haha

Reply:
hm, i'll just remove this for now then